nsxt/provider.go

```diff
@@ -528,6 +528,7 @@ func Provider() *schema.Provider {
 			"nsxt_vpc_static_route":                     resourceNsxtVpcStaticRoutes(),
 			"nsxt_policy_project_ip_address_allocation": resourceNsxtPolicyProjectIpAddressAllocation(),
 			"nsxt_vpc_dhcp_v4_static_binding":           resourceNsxtVpcSubnetDhcpV4StaticBindingConfig(),
+			"nsxt_vpc_attachment":                       resourceNsxtVpcAttachment(),
 		},
 
```

Comment:
I'd suggest this be the first in the lists of `nsxt_vpc_*` items (as well as ensure all listed resources and data sources are listed alphabetically.)